### ISSUE TITLE
Guild members with errored profiles causing Error 500

### ISSUE THREAD

**Issue by Skyrook** [danpaulson/swgoh.gg.public#58]
I've contacted Dan Paulson via a reddit thread here: https://www.reddit.com/r/SWGalaxyOfHeroes/comments/6z8pf3/swgohgg_guild_unit_search_platoon_helper/

Since, we've tried a few things- Hoarding energy to get into top 200 (got to #7 for two days) and when that didn't work after a few refreshes we disbanded our guild and formed a new guild.  Our new guild migrated the problem with it.  After some more troubleshooting, we discovered a member of our guild (Moe: 799-925-527) couldn't make his name on .gg so he made "Jorl/Moe" and we think the slash is the issue.
In finding this issue, we've also developed another with a member (Dboun: 332-298-588 - https://swgoh.gg/u/dboun/) who went from having a working profile, manually syncing, and not having an error 404 account as well. 

Old Guild: https://swgoh.gg/g/1494/law-eight/?stats=gp (Which was broken but now will open)
New Guild: https://swgoh.gg/g/30170/the-laws-of-eight/ (Which worked for the afternoon then errored)

Is there any way we could just get both of those accounts deleted so they can start new ones?  Trying to get them to make new accounts on swgoh.gg but assuming until the Jorl/Moe account is deleted we will continue to have this error and be unable to use your site!

Thanks!

**Comment by Skyrook** [comment]
Update, Dboun's profile worked itself out once it updated, it's just "Jorl/Moe" that we're requesting be deleted

**Comment by scoot1585** [comment]
Yea the "/" in names is definitely a problem.  The original registration should only allow numbers and letters, but people took advantage of the account renaming we allowed in settings which breaks a lot of stuff.  This is also why we disabled renaming in settings.  We don't really have a delete feature, but turning off sync and removing your player id and such is the same thing basically.

It looks like Dboun's account started arena syncing yesterday?  I am not getting a 404 error when I go to his profile.

And as for `Jorl/Moe`, I switched his username to `Jorl_Moe` to fix the issue he is most likely having.  Out of all of our accounts he was one of 6 people who had a slash in his username heh
